File: Projekat V2/Models/Paket.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Models
{
    [Table("Paket")]
    public class Paket
    {
        [Key]
        public int ID { get; set; }

        [Required]
        [MaxLength(50)]
        public String Posiljalac { get; set; }

        [Required]
        [MaxLength(50)]
        public string Primalac { get; set; }

        [Required]
        [Range(0.1, 100)]
        public float Masa { get; set; }

        [Required]
        public DateTime Datum_slanja { get; set; }

        public DateTime Datum_prijema { get; set; }

        public bool Lomljivo { get; set; }

        public bool Primljen { get; set; }

        public int CenaPosiljke { get; set; }

        [Required]
        [MaxLength(100)]
        public string Odrediste { get; set; }

        [JsonIgnore]
        public Posta Posta { get; set; }

        [JsonIgnore]
        public Vozilo Vozilo { get; set; }     
    }
}

File: Projekat V2/Client/Paket.js
export class Paket
{
    constructor(id, posiljalac, primalac, masa, datum_slanja, datum_prijema,
        lomljivo, primljen, cena_posiljke, adresa_prijema)
    {
        this.ID = id
        this.masa = masa;
        this.datum_slanja = datum_slanja;
        this.posiljalac = posiljalac;
        this.primalac = primalac;
        this.datum_prijema = datum_prijema;
        this.lomljivo = lomljivo;
        this.primljen = primljen;
        this.adresa_prijema = adresa_prijema;
        this.cena_posiljke = cena_posiljke
        this.container = null;
    }
}